plugin/evm/syncervm_client.go

```diff
@@ -15,6 +15,7 @@ import (
 	commonEng "github.com/ava-labs/avalanchego/snow/engine/common"
 	"github.com/ava-labs/avalanchego/snow/engine/snowman/block"
 	"github.com/ava-labs/avalanchego/vms/components/chain"
+
```

Comment:
nit: remove unnecessary diff

Reply:
it may be the editor playing tricks, but diff is still there